src/uct/base/uct_iface.h

```diff
@@ -173,6 +173,7 @@ typedef struct uct_base_iface {
         unsigned            num_alloc_methods;
         uct_alloc_method_t  alloc_methods[UCT_ALLOC_METHOD_LAST];
         ucs_log_level_t     failure_level;
+        size_t              max_iov;
```

Comment:
why do we need to save this in base interface?
1. if needed only for param check - can be put inside #ifdef
1. should be in rc_verbs_iface
1. should be unsigned rather than size_t